pyproject.toml

```diff
@@ -46,12 +46,11 @@ name = "ansys_sphinx_theme"
 
 [tool.flit.sdist]
 include = [
-    "src/ansys_sphinx_theme/layout.html",
-    "src/ansys_sphinx_theme/breadcrumbs.html",
-    "src/ansys_sphinx_theme/docs-navbar.html",
-    "src/ansys_sphinx_theme/theme.conf",
-    "src/ansys_sphinx_theme/_templates/",
-    "src/ansys_sphinx_theme/static/",
+    "src/ansys_sphinx_theme/theme/ansys_sphinx_theme/layout.html",
+    "src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/breadcrumbs.html",
+    "src/ansys_sphinx_theme/theme/ansys_sphinx_theme/theme.conf",
+    "src/ansys_sphinx_theme/theme/ansys_sphinx_theme/_templates/",
+    "src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/",
```

Comment:
I don't really like this folder structure... couldn't we reduce it to:

``src/ansys_sphinx_theme/theme/{files}``

Reply:
Okay, sure. As long as there is a reason for it, I'm fine with this approach =)